media/js/firefox/all/all-downloads-unified.js

```diff
@@ -172,6 +172,12 @@
         var currentPlatform = currentOptions.querySelector(
             '.c-selection-platform select'
         );
+        document
```

Comment:
Suggestion: can we add a code comment that explains what this is being used for?